docs/0x00_Description.md

```diff
@@ -1,15 +1,15 @@
 # Description
 
-As a Node.js developer, I know that for front-end developers who are new to the server field, it is easy to get lost when introducing Node.js:
+As a Node.js developer, I know it very well that for front-end developers who are new to the server field, it is easy to get lost when introducing Node.js:
```

Comment:
I know it very well that it is easy for front-end developers who are new to the server field to get lost when they are introducing Node.js: